### DISCUSSION TITLE
antd+tailwindcss按钮白色问题

### DISCUSSION THREAD

**Discussion by OnClickListener2048** [ant-design/ant-design#41858] [Q&A] [answered]
<Form
                            name="basic"
                            labelCol={{span: 6}}
                            wrapperCol={{span: 16}}
                            style={{maxWidth: 800}}
                            initialValues={{remember: true}}
                            onFinish={onFinish}
                            onFinishFailed={onFinishFailed}
                            autoComplete="on">
                            <Form.Item
                                label="用户名"
                                name="username"
                                rules={[{required: true, message: 'Please input your username!'}]}>
                                <Input/>
                            </Form.Item>

                            <Form.Item
                                label="密码"
                                name="password"
                                rules={[{required: true, message: 'Please input your password!'}]}>
                                <Input.Password/>
                            </Form.Item>

                            <Form.Item wrapperCol={{offset: 6, span: 16}}>
                                <Button type="primary" htmlType="submit">
                                    Submit
                                </Button>
                            </Form.Item>
                        </Form>
                        我使用tailwindcss和antd 5.4.2版本 将代码复制进去后发现按钮是白色 只有移动到按钮上面才会显示primary的颜色

**Comment by OnClickListener2048** [comment]
<img width="423" alt="image" src="https://user-images.githubusercontent.com/22124534/232698284-e695533d-ee6e-4dc5-8a08-0a3aaf21f07d.png">
<img width="437" alt="image" src="https://user-images.githubusercontent.com/22124534/232698369-f5ebc651-fb17-4fd0-a896-b5f9107212a5.png">

**Reply by OnClickListener2048** [reply]
上面是鼠标没有移动到按钮的时候 下面是鼠标移动到按钮的时候显示的样式

**Reply by csic21** [reply, marked as answer]
你直接看下是哪个 tailwindcss 的样式覆盖了

**Comment by Ray-56** [comment]
@OnClickListener2048 被基础按钮样式覆盖了。
目前我的做法是，自己写一份 base 将对应的地方注释了
![image](https://github.com/ant-design/ant-design/assets/17818179/4729baaa-ba92-46f9-8c13-ce37fa639d2d)
https://stackoverflow.com/questions/71783177/remove-specific-style-from-tailwind-base

**Comment by knight174** [comment]
<img width="756" height="197" alt="image" src="https://github.com/user-attachments/assets/69366069-32b8-4c5b-8626-c14cdaf79c9d" />

升级到 3.4.3 版本有效！
https://stackoverflow.com/questions/71783177/remove-specific-style-from-tailwind-base
